src/index.html

```diff
@@ -54,16 +54,15 @@
         This application requires JavaScript to run. Please enable JavaScript in
         your browser.
     </noscript>
-    <div id="header">
-      <span>Daily chat + reaction recording with VCS</span><span>powered by <a href="https://docs.daily.co">//daily</a></span>
-    </div>
     <div id="container">
         <div id="lobby" class="lobby">
             <h1>Daily chat + reaction recording with VCS</h1>
             <p>This demo uses <a href="https://docs.daily.co/reference/daily-js">Daily's Client SDK for JavaScript</a> and <a href="https://docs.daily.co/reference/vcs">Daily's Video Component System</a> to embed chat and emoji reaction elements into a cloud recording.</a></p>
             <p>To use it, specify a Daily room with <a href="https://docs.daily.co/guides/products/live-streaming-recording/recording-calls-with-the-daily-api#cloud">cloud recording enabled</a>.</p>
             <form id="joinForm">
+              <label>Daily room URL</label>
```

Comment:
```suggestion
              <label for="url">Daily room URL</label>
```
to bind the label and input elements. (Clicking the label will highlight the input this way too)